docs/release_notes/release_notes.md

```diff
@@ -11,8 +11,67 @@ has_children: true
+
+    * #### Device Inventory dashboard
+        * On **Possible Merge UUIDs (Devices) in Device Inventory** panel, added possible combinations of devices that can be merged based on hostname matching.
+        * Added forwarder type information for the splunk devices to the **Device Inventory** table.
+        * Added more details of the splunk devices like os, host, version, forwarder type to compare with other devices on **Possible Merge UUIDs (Devices) in Device Inventory** panel.
+        * Added splunk devices panel on Intelligence dashboard.
+
+    * #### Network Telemetry dashboard
+        * Added filter to search port number.
+        * Added reporting device IP information as well as vulnerability information for the vulnerable traffic.
+
+    * Enhanced the **O365 - OneDrive or SharePoint File Sharing with External User** and **O365 - OneDrive or SharePoint Link Accessed By External User** alerts to get accurate results.
+
+    * Converted the detection time format as per the local timezone for **Windows Defender - Malware Detected** alert.
+
+    * Filtered out events having the user "Not Available" from **Authentication - Bruteforce Attempt for a User** and **Authentication - Bruteforce Attempt from a Source** alerts.
+
+    * Filtered out compliance related events from vulnerability alerts like **Asset Inventory - Vulnerability Lookup Gen** & **Device Inventory - Tenable Vuln**.
+
+    * Added location information to the **Authentication activities** & **Radius Authentication activities** panels on the Intelligence dashboard.
+
+    * Reduced the severity to "low" for **Authentication - Excessive Failed VPN Logins for a User** alert.
+
+    * Reduced the severity of the **Email - Hourly Increase In Emails Over Baseline** alert.
+
+    * Updated the upperbound value calculation for the **Network Compromise - Calculate UpperBound for Spike in Network Traffic** and **Network Compromise - Calculate UpperBound for Spike in Outbound Network Traffic** alerts.
+
+    * Updated Splunk-python-sdk to the latest version.
 
 
 ## Upgrade Guide from 4.8.0 to 4.9.0
 
-* Check all the cyences alerts and if the user wants to keep it enable/disable permanently then perform the steps mentioned [here]({{ site.baseurl }}/install_configure/alert_configuration/#how-to-disable-the-alert-state-changes-performed-from-the-cyences-setup-page)
\ No newline at end of file
+* To onboard the nessus pro logs, configure the [Nessus Professional Add-On for Splunk](https://splunkbase.splunk.com/app/7464/)
+
+* Check all the cyences alerts and if the user wants to keep it enabled/disabled permanently then perform the steps mentioned [here]({{ site.baseurl }}/install_configure/alert_configuration/#how-to-disable-the-alert-state-changes-performed-from-the-cyences-setup-page)
+
+* To onboard the sophos endpoint logs, configure the sophos endpoint input on [Sophos Central Add-on for Splunk](https://splunkbase.splunk.com/app/6186/)
```

Comment:
After upgrade collection of Sophos Endpoint logs are required to keep the Device inventory working as expected.